lib/sqlalchemy/sql/base.py

```diff
@@ -1641,6 +1641,9 @@ def compare(self, other: ColumnCollection[Any, Any]) -> bool:
     def __eq__(self, other: Any) -> bool:
         return self.compare(other)
 
+    @overload
+    def get(self, key: str, default: _COL_co) -> _COL_co: ...
+
     def get(
```

Comment:
I think you need to provide also this overload, otherwise default becomes required. (Also iirc at least two overload are needed or the type checker complaints

Reply:
Done.